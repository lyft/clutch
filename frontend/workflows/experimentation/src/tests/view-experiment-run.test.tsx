import React from "react";
import { shallow } from "enzyme";

import ViewExperimentRun from "../view-experiment-run";

jest.mock("react-router-dom", () => {
  return {
    ...jest.requireActual("react-router-dom"),
    useNavigate: jest.fn(),
  };
});

describe("View Experiment Run workflow", () => {
  let component;

  beforeAll(() => {
    component = shallow(<ViewExperimentRun />);
  });

  it("renders correctly", () => {
    expect(component.debug()).toMatchSnapshot();
  });
});
