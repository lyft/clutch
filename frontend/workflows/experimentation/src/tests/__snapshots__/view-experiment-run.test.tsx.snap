// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Experiment Run workflow renders correctly 1`] = `
"<PageLayout heading={[undefined]} error=\\"\\">
  <styled.form />
</PageLayout>"
`;
